ext/sqlite3/sqlite3.c

```diff
@@ -1057,7 +1039,8 @@ PHP_METHOD(SQLite3, createCollation)
 	if (sqlite3_create_collation(db_obj->db, collation_name, SQLITE_UTF8, collation, php_sqlite3_callback_compare) == SQLITE_OK) {
 		collation->collation_name = estrdup(collation_name);
 
-		ZVAL_COPY(&collation->cmp_func, &fci.function_name);
+		zend_fcc_addref(&fcc);
+		memcpy(&collation->cmp_func, &fcc, sizeof(zend_fcall_info_cache));
```

Comment:
Wouldn't it make sense to introduce a macro/inline function for this?

Reply:
That make sense, how about ``zend_fcc_copy()``? Or should ``dup`` be used instead?

Reply:
`dup` sounds like a good choice!